docker/app.dockerfile

```diff
@@ -1,13 +1,22 @@
-FROM php:8.1-fpm AS php-build
+FROM php:8.3-fpm AS php-build
 
 RUN apt-get update && apt-get install -y  \
-    libzip-dev \
+    libzip-dev git libmagickwand-dev libzip-dev \
     --no-install-recommends \
-    && docker-php-ext-install pdo_mysql zip exif
+    && docker-php-ext-install pdo_mysql zip exif && \
```

Comment:
PHP 8.3 Imagick PECL install is currently broken. It has to be built from source. I am not sure if I did this in the right layer, and I am not sure what Imagick is used for, so I am not sure how to test if it still works.

Reply:
It's always Imagick and/or gd. 

I think this was used for the image preview on the advance. I am not 100% positive though.